source/libs/cache.ts

```diff
@@ -45,30 +45,35 @@ export function set<TValue extends any = any>(key: string, value: TValue, expira
 }
 
 /* Accept messages in background page */
-if (!browser.runtime.getBackgroundPage) {
-	browser.runtime.onMessage.addListener((request: CacheRequest, _sender, sendResponse) => {
```

Comment:
Here also it would be good to replace `sendResponse` with a Promise, because what I think broke the feature was the polyfill dropping the `sendResponse` callback
<img width="686" alt="" src="https://user-images.githubusercontent.com/1402241/58525549-b16e8f80-81fe-11e9-99b1-1d975fbaff56.png">

Docs here: https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/runtime/onMessage

> return a `Promise` from the event listener, and resolve when you have the response (or reject it in case of an error).